virtual/libelf/libelf-4.ebuild

```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Virtual for libelf.so.1 provider dev-libs/elfoolchain"
+SLOT="0/1"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-libs/elftoolchain"
```

Comment:
Why is this the only provider now?

We should revbump the existing one and add || ( this ... existing ), right?

Reply:
Hm, `-2` has only libelf, and `-3` only elfutils so I was following that

Reply:
Ah I found some discussion in https://bugs.gentoo.org/643802 about this

Reply:
I need to read that bug properly but need @SoapGentoo's input here too.

Reply:
Yeah, I've removed the virtual commit for now, we can do it later as far as I'm concerned